src/tools/clippy/tests/ui/extra_unused_lifetimes.stderr

```diff
@@ -6,12 +6,6 @@ LL | fn unused_lt<'a>(x: u8) {}
    |
    = note: `-D clippy::extra-unused-lifetimes` implied by `-D warnings`
 
-error: this lifetime isn't used in the function definition
```

Comment:
@flip1995 I mean this change, and the one below. I don't understand how clippy use to account for lifetime uses in bounds and not where clauses.

Reply:
I can't really tell what would cause this from just looking at the Clippy changes. I'll have to review and understand the other changes to rustc in this PR. This might take me a few days until I get to that though.